schedulescaling.tf

```diff
@@ -0,0 +1,59 @@
+resource "aws_appautoscaling_scheduled_action" "table_read_schedule" {
+  for_each = var.create_table && var.autoscaling_enabled && length(var.schedule_scaling_read) > 0 ? { for k, v in var.schedule_scaling_read : k => v } : {}
+
+  name = "DynamoDBReadCapacityUtilization-${replace(aws_appautoscaling_target.table_read[0].resource_id, "/", "-")}-${each.key}"
+
+  resource_id        = aws_appautoscaling_target.table_read[0].resource_id
+  scalable_dimension = aws_appautoscaling_target.table_read[0].scalable_dimension
+  service_namespace  = aws_appautoscaling_target.table_read[0].service_namespace
+
+  schedule = each.value["schedule"]
+
+  scalable_target_action {
+    max_capacity = each.value["max_capacity"]
+    min_capacity = each.value["min_capacity"]
+  }
+}
+
+module "index_read_schedule" {
+  for_each = var.create_table && length(var.schedule_scaling_indexes_read) > 0 ? var.autoscaling_indexes : {}
```

Comment:
Looks pretty good, but please remove `module/module` and implement the same logic in the existing `autoscaling.tf`